### PR TITLE
Added extra context in send invitation

### PR DESCRIPTION
**Pull request by joshjo**
When you send an invitation to the email, you could need to add extra information in the email. So then I update the `send_invitation` function in order to append the `kwargs` to the context used in the email.

### CODE REVIEW

**Comment by pandabearcoder** [comment]
This looks good, when will this be merged. I bet a lot of people need this

**Comment by bee-keeper** [comment]
Thanks for this.  Apologies it took so long to merge.

File: invitations/models.py [hunk 1]
```diff
@@ -43,19 +43,18 @@ def key_expired(self):
         return expiration_date <= timezone.now()
 
     def send_invitation(self, request, **kwargs):
-        current_site = (kwargs['site'] if 'site' in kwargs
-                        else Site.objects.get_current())
+        current_site = kwargs.pop('site', Site.objects.get_current())
```
Comment (pandabearcoder) [inline invitations/models.py]: I suggest using kwargs.get here 
```
current_site = kwargs.get('site', Site.objects.get_current())
```

**Comment by pandabearcoder** [comment]
Yay thanks!

**Pull request merged**